test/d-b-n_smoke/arrays.html

```diff
@@ -162,6 +162,14 @@
                     done();
                 }, 100);
             });
+
+            /** This should be used with spy on patchsent event */
+            function expectPatchSent(operation) {
```

Comment:
Do you mean custom chai assertion (`expect(operationObject(...)).to.be.sent`)? I think it's pretty readable now, but it also uses `patchSentEventCallback`, which is defined locally (so extracting it would actually hurt readability)